conformance/reports/v1.1.0/nginxinc-nginx-gateway-fabric/README.md

```diff
@@ -0,0 +1,23 @@
+# Nginxinc NGINX Gateway Fabric
+
+## Table of Contents
+
+| API channel | Implementation version                                                         | Mode | Report                       |
+|-------------|--------------------------------------------------------------------------------|------|------------------------------|
+| x           | [v1.3.0](https://github.com/nginxinc/nginx-gateway-fabric/releases/tag/v1.3.0) | x    | [link](./v1.3.0-report.yaml) |
```

Comment:
I think the name of this report should change per https://github.com/kubernetes-sigs/gateway-api/pull/3148.

/cc @mlavacca

Reply:
Yep, here is the naming we must follow for reports with Gateway API version >=1.1 https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/README.md

Reply:
I can rename